vtv-shows-app/src/app/page.module.css

```diff
@@ -0,0 +1,72 @@
+.main {
+    font-family: Arial, sans-serif;
+    background-color: #3D087B;
+    color: #FFF;
+    display: flex;
+    justify-content: center;
+    align-items: flex-start;
+    height: 100vh;
+    gap: 20px;
+    padding: 20px; /* Add some padding at the top */
+}
+
+.container {
+    max-width: 600px;
+    margin: 20px;
+    padding: 20px;
+    background-color: #2D0A55;
+    border-radius: 10px;
+    text-align: center;
+}
+
+.show-info img {
```

Comment:
You are not using this css. You should remove most if not all content from this file for that reason. Like we did in the lecture.